provider/cmd/pulumi-resource-cloudflare/schema.json

```diff
@@ -427,15 +434,17 @@
                     }
                 },
                 "identityProviderId": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The ID of the Azure Identity provider.\n"
                 }
             },
             "type": "object"
         },
         "cloudflare:index/AccessGroupExcludeOkta:AccessGroupExcludeOkta": {
             "properties": {
                 "identityProviderId": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "The ID of the Azure Identity provider.\n"
```

Comment:
Why does this refer to azure?

Reply:
This resource interacts with a bunch of other account sources, on of which is azure. This should be for `okta`, but the parse has mis-sourced the description.

My guess is that this is an existing bug about how an "empty" description should be interpreted.